cmd/clusterctl/client/cluster/cert_manager_test.go

```diff
@@ -35,21 +34,24 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	clusterctlv1 "sigs.k8s.io/cluster-api/cmd/clusterctl/api/v1alpha3"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/client/config"
-	manifests "sigs.k8s.io/cluster-api/cmd/clusterctl/config"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/internal/scheme"
 	"sigs.k8s.io/cluster-api/cmd/clusterctl/internal/test"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	testHash    = "af8c08a8eb65d102ba98889a89f4ad1d3db5d302edb5b8f8f3e69bb992faa211"
+	testVersion = "v0.16.1"
```

Comment:
I would really like to find a way for us for not being forced to maintain those values in the code, but tat the same time for being sure we can derive those values from the embedded manifest whenever a new manifest is added

Reply:
yes, I think something mentioned about this in the original issue
given this only occurs in test ,is it mandatory ? or any suggestion on how to get it? thanks